route/build/src/tracer.f90

```diff
@@ -0,0 +1,364 @@
+MODULE tracer_module
+
+! solve 1-D advection(-diffusion) equation from upstream to downstream
```

Comment:
I think this is another place to mention that this only setup for solute constituents.

Reply:
This comment has been revised